src/ophys_etl/transforms/suite2p_container.def

```diff
@@ -0,0 +1,44 @@
+Bootstrap: docker
```

Comment:
In general maybe take a read through: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/

I suspect a lot of the best practices should apply to singularity files as well.

Reply:
i'll take a look. 
also [here](https://sylabs.io/guides/3.5/user-guide/definition_files.html#best-practices-for-build-recipes)

Reply:
Added `%help` section and now conforms to these [best practices](https://sylabs.io/guides/3.5/user-guide/definition_files.html#best-practices-for-build-recipes).